content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -28,6 +28,12 @@ a Pod or Container. Security context settings include:
 
 * AllowPrivilegeEscalation: Controls whether a process can gain more privileges than its parent process. This bool directly controls whether the [`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt) flag gets set on the container process. AllowPrivilegeEscalation is true always when the container is: 1) run as Privileged OR 2) has `CAP_SYS_ADMIN`.
 
+* readOnlyRootFilesystem: Mounts the container's root filesystem as read-only.
+
+The above bullets are not a complete set of security context settings -- please see 
```

Comment:
@JayBeale , What do you think about updating lines 33-35:

For a complete list of security context settings, see the Kubernetes [SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core) API.

Reply:
That sounds perfect @kbhawkey!